views/page/course/course-overview.html

```diff
@@ -78,10 +79,45 @@ <h2 class="govuk-heading-m">Course audiences</h2>
                             }
                         ] %}
                         {{ menu("Actions", actions) }}
+
+                        {% set linkActions = [
+                            {
+                                link: courseUrl,
+                                text: 'View this course on Civil Service Learning',
```

Comment:
Could we add 'opens in a new tab'? Maybe in some hidden text?

Reply:
Good idea

Reply:
Either or is fine, I think - see what Sam thinks when we demo. LGTM otherwise